src/main/java/vg/civcraft/mc/civmodcore/command/PlayerCommand.java

```diff
@@ -2,50 +2,83 @@
 
 import org.bukkit.Bukkit;
 import org.bukkit.entity.Player;
+
+import vg.civcraft.mc.civmodcore.util.TextUtil;
+
+import org.bukkit.command.CommandSender;
 import org.bukkit.command.PluginCommand;
 
 public abstract class PlayerCommand implements Command {
 
 	private String name = "";
-
 	private String description = "";
-
 	private String usage = "";
-
 	private String identifier = "";
-
 	private int min = 0;
-
 	private int max = 0;
+	private boolean senderMustBePlayer = false;
+	private boolean errorOnTooManyArgs = true;
+	private CommandSender sender;
+	private String[] args;
```

Comment:
Saving the sender makes sense if you need it for the utility methods, but why the arguments?